engine/src/test/resources/testAppPackage/mydtapp/src/main/java/com/example/mydtapp/Application.java

```diff
@@ -37,6 +37,8 @@ public void populateDAG(DAG dag, Configuration conf)
 
     RandomNumberGenerator rand = dag.addOperator("rand", RandomNumberGenerator.class);
     StdoutOperator stdout = dag.addOperator("stdout", new StdoutOperator());
+    // This module will be added to dag for testing purpose but will not be connected in a dag.
+    dag.addModule("testModule", com.example.mydtapp.TestModule.class);
```

Comment:
@sandeshh This application is a part of the setup for tests in AppPackageTest. If the code has the module as part of DAG and fix is not provided, tests will fail. But even if I remove the module from tests and with a fix, tests will run as they supposed to. So the module is required to make sure that properties are made available to Custom Modules.